js/components/frame.js

```diff
@@ -54,12 +54,6 @@ class Frame extends ImmutableComponent {
 
   componentDidMount () {
     this.updateWebview()
-    // forward postMessage events from webview to webContents
-    window.addEventListener('message', function (event) {
-      if (this.webview.getAttribute('src').match(event.origin)) {
```

Comment:
document.domain will be set to blank and also I think origin for a file URL will always return file://